lib/stamp/emitters/two_digits_only.rb

```diff
@@ -0,0 +1,20 @@
+module Stamp
+  module Emitters
+    # Emits the given field as a two-digit number with a leading
+    # zero if necessary.
+    class TwoDigitsOnly
+      attr_reader :field, :add_zero
+
+      # @param [field] the field to be formatted (e.g. +:year+)
+      def initialize(field, add_zero = false)
```

Comment:
```ruby
def initialize(field, leading_zero: false)
```

I prefer a keyword argument over an unnamed boolean. I think it's time to drop support for Ruby < 2.0, so we don't need a soft options hash.